src/components/Register/index.jsx

```diff
@@ -0,0 +1,252 @@
+import { Alert, Button, Checkbox, Input } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { useState } from "react";
+import { validateEmail } from "../../functions";
+import { useTranslation } from "react-i18next";
+
+const Register = (props) => {
+  const [t] = useTranslation();
+  const maxLenghtPassword = 8;
+  const [registerInformation, setRegisterInformation] = useState({
+    fullName: "",
+    age: "",
+    email: "",
+    password: "",
+    agree: false,
+  });
+  const [errors, setErrors] = useState({});
+  const [termsOpen, setTermsOpen] = useState(false);
+  const handleChange = (key, value) => {
+    let newValues = Object.assign({}, registerInformation);
+    newValues[key] = value;
+    setRegisterInformation(newValues);
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    let { fullName, email, age, password, agree } = registerInformation;
+    let errors = {};
```

Comment:
`errors` is a state. If you want to set it you should use `setErrors`. But it is already empty at first.

Reply:
if you want to make copy of existing errors use `const copyErrors = {...errors}`.

`const `should be used

Reply:
If this is working reason must be scope. This is in lower scope and names are not clashing. But still you should be using different names. Because as you can see from my comments, same names cause confusion.